<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Форма</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="status">
     <property name="styleSheet">
      <string notr="true">font: 25 50pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>%status%</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ваш счёт составляет:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ad_bal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>%balance%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="company_dept">
     <property name="text">
      <string>%company_dept%</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>235</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="address">
       <property name="text">
        <string>%address%</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="back_b">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(53, 52, 77);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Назад к форме входа</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>back_b</sender>
   <signal>clicked()</signal>
   <receiver>admin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
